plugin/db/driver_test.go

```diff
@@ -4,22 +4,23 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/bmizerany/assert"
```

Comment:
Should be github.com/stretchr/testify/assert

Reply:
Done.